tests/support/markdownthemewitiko_markdown_test.sty

```diff
@@ -278,6 +278,10 @@
       \TYPE{superscript:           #1}},
     subscript = {%
       \TYPE{subscript:             #1}},
+    displayMath = {%
+      \TYPE{displayMath}},
+    inlineMath = {%
+      \TYPE{inlineMath}},
```

Comment:
Any reason why we cannot display `#1` in the output?

Reply:
```suggestion
    displayMath = {%
      \TYPE{displayMath:           #1}},
    inlineMath = {%
      \TYPE{inlineMath:            #1}},
```

Reply:
Well, this is produced when including the argument in the unit-tests.

Plain TeX:
```
*** 1,5 ****
  documentBegin
  inlineMath: E=mc^2
  interblockSeparator
! displayMath: \hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx
  documentEnd
--- 1,5 ----
  documentBegin
  inlineMath: E=mc^2
  interblockSeparator
! displayMath: \mathaccent "705E {f} \left  ( \xi   \right  )= \intop \nolimits _{-\infty }^{\infty } f\left  ( x  \right  ) e^{-i2\pi  \xi  x} dx
  documentEnd
```

LaTeX:
```
*** 1,5 ****
  documentBegin
  inlineMath: E=mc^2
  interblockSeparator
! displayMath: \hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx
  documentEnd
--- 1,5 ----
  documentBegin
  inlineMath: E=mc^2
  interblockSeparator
! displayMath: \protect \mathaccent "705E\relax {f} \left  ( \xi   \right  )= \protect \intop \nolimits _{-\infty }^{\infty } f\left  ( x  \right  ) e^{-i2\pi  \xi  x} dx
  documentEnd
```

ConTeXt:
```
*** 1,5 ****
  documentBegin
  inlineMath: E=mc^2
  interblockSeparator
! displayMath: \hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx
  documentEnd
--- 1,5 ----
  documentBegin
  inlineMath: E=mc^2
  interblockSeparator
! displayMath: \hat {f} \left  ( \xi   \right  )= \int _{-\infty }^{\infty } f\left  ( x  \right  ) e^{-i2\pi  \xi  x} dx
  documentEnd
```

Reply:
At the beginning of your unit test, you can undefine all commands that you use inside inline math to prevent expansion. Like this:
```
\let\int=\relax
```

Reply:
What about the additional space after each macro?

For input `$$\hat{f}$$`, should the unit test be  `displayMath: \hat{f}` or  `displayMath: \hat {f}` ?

Reply:
TeX will always include a space after the command in the output. Functionally, `\hat{f}` and `\hat {f}` are also equivalent, since the input processor of TeX will gobble spaces after a command.